src/test/java/org/betonquest/betonquest/modules/logger/custom/HistoryLogHandlerTest.java

```diff
@@ -24,10 +25,17 @@
      */
     private static final LocalDateTime FIXED_TIME = LocalDateTime.of(2022, 1, 1, 0, 0);
 
+    /**
+     * The mocked plugin instance.
+     */
+    private final Plugin plugin;
+
     /**
      * Default constructor.
      */
     public HistoryLogHandlerTest() {
+        plugin = mock(Plugin.class);
```

Comment:
The Plugin mock should be reset() after every test.

```java
@BeforeEach
void prepareMocks() {
    when(plugin.getName()).thenReturn("BetonQuest");
}

@AfterEach
void resetMocks() {
    reset(server);
}
```

An alternative would be to use the `@Mock` annotation:
https://javadoc.io/static/org.mockito/mockito-core/4.3.1/org/mockito/Mockito.html#mock_annotation